src/page-modules/departures/details/index.tsx

```diff
@@ -55,7 +56,16 @@ export function DeparturesDetails({
         )}
       </div>
 
-      <div className={style.mapContainer}></div>
+      <div className={style.mapContainer}>
+        <Map
+          position={[
```

Comment:
Is it possible to pass this as an object with named properties instead of an array? I'm in the same boat as @mikaelbr, not knowing which comes first.